### PR TITLE
1Flow iOS doc update

### PR DESCRIPTION
**Pull request by rohantryskybox**

### CODE REVIEW

File: src/connections/destinations/catalog/test-flow/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+This destination is maintained by 1Flow. For any issues with the destination, [contact their Support team](mailto:support@1flow.app).
+
+## Getting started
+
+1. From the Segment web app, click **Catalog**, then click **1Flow**.
+2. Find the Destinations Actions item in the left navigation, and click it.
+3. Click **Configure 1Flow**.
+4. Select an existing Source to connect to 1Flow (Actions).
+5. Copy 1Flow project key and paste it into the Connection Settings in Segment.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/test-flow/index.md:17]: Hi @rohantryskybox - where could a user find the 1Flow project key? Could we add a step or two here explaining how you could find the key in 1Flow?

Comment (rohantryskybox) [reply src/connections/destinations/catalog/test-flow/index.md]: Done

File: src/connections/destinations/catalog/test-flow/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+title: 1Flow (Actions) Destination
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/test-flow/index.md:2]:
```suggestion
title: 1Flow Mobile Plugin Destination
id: 64dd07c1fed86b6866cd93f5
```

File: src/connections/destinations/catalog/test-flow/index.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+This destination is maintained by 1Flow. For any issues with the destination, [contact their Support team](mailto:support@1flow.app).
+
+## Getting started
+
+1. From the Segment web app, click **Catalog**, then click **1Flow**.
+2. Find the Destinations Actions item in the left navigation, and click it.
+3. Click **Configure 1Flow**.
+4. Select an existing Source to connect to 1Flow (Actions).
+5. Go to 1flow.ai -> Settings -> Project Settings -> Copy 1Flow project key and paste it into the Connection Settings in Segment.
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/test-flow/index.md:17]:
```suggestion
1. From the Segment web app, click **Catalog**, then search for **1Flow Mobile Plugin**.
2. Click **Add Destination**.
4. Select an existing Source to connect to 1Flow Mobile Plugin.
5. Go to 1flow.ai -> Settings -> Project Settings, copy the 1Flow project key, and paste it into the Destination Settings in Segment.
6. Depending on the mobile source you’ve selected, include 1Flow's library by adding the following lines to your dependency configuration.
```

File: src/connections/destinations/catalog/test-flow/index.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/test-flow/index.md:18]: Add installation instructions for each library, see example documentation here: 

- Braze iOS installation: https://segment.com/docs/connections/destinations/catalog/braze/#ios
- Braze Android installation: https://segment.com/docs/connections/destinations/catalog/braze/#android

```suggestion
### iOS

1. Add the 1Flow Segment Pod to your `Podfile`:

    ```
    pod '1Flow-Segment'
    ```

(add other relevant instructions)

### Android


1. In the top-level project `build.gradle` add the following as a repository in *all projects > repositories*.

    ```js
    maven { url "http://appboy.github.io/1flow-android-sdk/sdk" }
    maven { url "http://appboy.github.io/1flow-segment-android/sdk" }
    ```

(add other relevant instructions)
```

File: src/connections/destinations/catalog/test-flow/index.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+---
+title: 1Flow (Actions) Destination
+---
+
+[1Flow](https://1flow.ai/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) is a leading in-app user survey and messaging platform for Mobile app and SaaS businesses.
+
+Using 1Flow, you can reach users _in-the-moment_ while they are interacting with your website or application, to collect highly contextual user insights that help you improve your product offering and customer experience
+
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/test-flow/index.md:8]:
```suggestion
The 1Flow Mobile Plugin Destination is open-source and available on GitHub. You can view these repositories here: 

- [iOS](github url){:target="_blank"} 
- [Android](github url){:target="_blank"}
```

**Review by tcgilbert — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/test-flow/index.md (hunk 2 above)
Comment (rohan1flow) [reply src/connections/destinations/catalog/test-flow/index.md]: Done. Thanks for suggestion

File: src/connections/destinations/catalog/test-flow/index.md (hunk 3 above)
Comment (rohan1flow) [reply src/connections/destinations/catalog/test-flow/index.md]: Done

File: src/connections/destinations/catalog/test-flow/index.md (hunk 4 above)
Comment (rohan1flow) [reply src/connections/destinations/catalog/test-flow/index.md:47]: Done. Also added setup instructions

File: src/connections/destinations/catalog/test-flow/index.md (hunk 5 above)
Comment (rohan1flow) [reply src/connections/destinations/catalog/test-flow/index.md:8]: Done

File: src/connections/destinations/catalog/test-flow/index.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
+---
+title: 1Flow Mobile Plugin Destination
+---
+
+[1Flow](https://1flow.ai/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) is a leading in-app user survey and messaging platform for Mobile app and SaaS businesses.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/test-flow/index.md:5]:
```suggestion
[1Flow](https://1flow.ai/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} is a leading in-app user survey and messaging platform for Mobile app and SaaS businesses.
```

File: src/connections/destinations/catalog/test-flow/index.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
+---
+title: 1Flow Mobile Plugin Destination
```
Comment (forstisabella) [inline src/connections/destinations/catalog/test-flow/index.md:2]:
```suggestion
title: 1Flow Mobile Plugin Destination
id: 64dd07c1fed86b6866cd93f5
beta: true
hidden: true
```

**Review by forstisabella — APPROVED** [review]
Approved and g2g pending two comments!

**Comment by rohan1flow** [comment]
> Approved and g2g pending two comments!

Addressed both. Ready if you want to take a look

**Comment by forstisabella** [comment]
lgtm! I'll wait for one more review from @tcgilbert before we merge

**Review by tcgilbert — APPROVED** [review]

**Pull request merged**